src/main/java/com/faforever/api/config/FafApiProperties.java

```diff
@@ -250,4 +251,14 @@
   public static class Tutorial {
     private String thumbnailUrlFormat;
   }
+
+  @Data
+  public static class ForgedAllianceExe {
```

Comment:
This whole block should be merged with the existing class `FafApiProperties.Deployment` (contains the `forgedAllianceExePath` already.

Reply:
Do I understand what you mean right?

```
  @Data
  public static class Deployment {
    private String featuredModsTargetDirectory;
    private String repositoriesDirectory;
    private String filesDirectoryFormat = "updates_%s_files";
    private String forgedAllianceExePath;
    private String allowedExeExtension = "exe";
    private String forgedAllianceBetaExePath;
    private String forgedAllianceDevelopExePath;

  }
```

Reply:
Yes. That looks better.